helpers/TimeLeft.ts

```diff
@@ -0,0 +1,3 @@
+const today = new Date();
+export const time =
+  today.getHours() + ":" + today.getMinutes() + ":" + today.getSeconds();
```

Comment:
This function is still hardcoded. This is not how you implement a countdown.

Please check for a better implementation.